From b64fcc414316592968f181c85447cfd01d1e461e Mon Sep 17 00:00:00 2001
From: Yi Fan Yu <yifan.yu@windriver.com>
Date: Thu, 15 Apr 2021 13:48:19 -0400
Subject: [PATCH] *.py: s/python/python3/ (exclude tests)

As stated by https://github.com/syslog-ng/syslog-ng/pull/3603
python2 is EOL.

Fix all shebangs calling python instead of python3
except the tests.

(correcting lib/merge-grammar.py)
Signed-off-by: Joe Slater <joe.slater@windriver.com>
(adding the rest)
Upstream-Status: Submitted [https://github.com/syslog-ng/syslog-ng/pull/3647]

Signed-off-by: Yi Fan Yu <yifan.yu@windriver.com>
---
 contrib/scripts/config-graph-json-to-dot.py | 2 +-
 lib/merge-grammar.py                        | 2 +-
 modules/python/pylib/setup.py               | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/contrib/scripts/config-graph-json-to-dot.py b/contrib/scripts/config-graph-json-to-dot.py
index 4955c81..0351a9a 100755
--- a/contrib/scripts/config-graph-json-to-dot.py
+++ b/contrib/scripts/config-graph-json-to-dot.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import json, sys
 
 j = None
diff --git a/lib/merge-grammar.py b/lib/merge-grammar.py
index 7313ff5..459712d 100755
--- a/lib/merge-grammar.py
+++ b/lib/merge-grammar.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #############################################################################
 # Copyright (c) 2010-2017 Balabit
 #
diff --git a/modules/python/pylib/setup.py b/modules/python/pylib/setup.py
index 23bb5cc..a2fa05e 100755
--- a/modules/python/pylib/setup.py
+++ b/modules/python/pylib/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #############################################################################
 # Copyright (c) 2015-2016 Balabit
 #
